fly.toml

```diff
@@ -12,9 +12,6 @@ auto_rollback = true
 source = "data"
 destination = "/data"
 
-[deploy]
-release_command = "node ./other/sentry-create-release"
```

Comment:
as far as I understand, this file has been superseded by use of the `sentryVitePlugin`, which does the release and sourcemaps upload and deletion for you